src/objdictgen/node.py

```diff
@@ -79,14 +80,39 @@ def StringFormat(text, idx, sub):  # pylint: disable=unused-argument
     if result:
         fmt = result.groups()
         try:
-            log.debug("EVAL StringFormat(): '%s'" % (fmt[1],))
-            return fmt[0] % eval(fmt[1])  # FIXME: Using eval is not safe
+            args = fmt[1].split(',')
```

Comment:
While it's great to get rid of evaluate (we need that), this part belongs in a separate PR.

Reply:
I have partially reverted the commit with these changes (https://github.com/Laerdal/python-objdictgen/pull/2/commits/9168780cb4e4de4881f4b2edf8e01b83e142ee4c).

I will move these changes to a separate branch/PR as requested.